advanced/Scripts/version.sh

```diff
@@ -97,22 +112,48 @@ getRemoteVersion(){
     return 0
 }
 
+getLocalBranch(){
+    # Get the checked out branch of the local directory
+    local directory="${1}"
+    local branch
+
+     # Local FTL btranch is stored in /etc/pihole/ftlbranch
+    if [[ "$1" == "FTL" ]]; then
+        branch=$(cat /etc/pihole/ftlbranch)
+    else
+        cd "${directory}" 2> /dev/null || { echo "${DEFAULT}"; return 1; }
+        branch=$(git rev-parse --abbrev-ref HEAD || echo "$DEFAULT")
+    fi
+    if [[ ! "${branch}" =~ ^v ]]; then
+        if [[ "${branch}" == "master" ]]; then
+            echo ""
+        else
+            echo "${branch} "
+        fi
+    elif [[ "${branch}" == "${DEFAULT}" ]]; then
+        echo "ERROR"
+        return 1
+    else
+        echo "No Checked out branch"
```

Comment:
Could this result in
```
Pi-hole version is No Checked out branchv4.3.5-460-g413fa94 (Latest: v4.4)
```
(also note the missing space between branch and v4.3.5)?

I'm not sure how/if we could reach this `else` condition.

Reply:
We can probably lose the `else` section... the function was a copy-paste, and then tweak of `getLocalVersion()`